src/main/java/uk/gov/pay/ledger/transaction/dao/TransactionDao.java

```diff
@@ -29,19 +29,20 @@
 
     private static final String UPSERT_STRING =
             "INSERT INTO transaction(" +
-                "external_id,gateway_account_id,amount,description,reference,state,email,cardholder_name," +
+                "external_id,parent_transaction_id,gateway_account_id,amount,description,reference,state,email,cardholder_name," +
                 "external_metadata,created_date,transaction_details,event_count,card_brand, " +
                 "last_digits_card_number,first_digits_card_number,net_amount,total_amount,fee,type" +
             ") " +
             "VALUES (" +
-                ":externalId,:gatewayAccountId,:amount,:description,:reference,:state,:email,:cardholderName," +
+                ":externalId,:parentExternalId,:gatewayAccountId,:amount,:description,:reference,:state,:email,:cardholderName," +
```

Comment:
Is there a reason both `parent_transaction_id` and `parentExternalId` are used? Agreed that getting external in the name would be a good thing but we should keep it consistent. 

`parent_external_id`
`parent_transaction_external_id`

Reply:
Renamed in https://github.com/alphagov/pay-ledger/pull/147